app.js

```diff
@@ -0,0 +1,66 @@
+function getPlayerMove() {
+  //Gets player input as string
+  var playerinput = prompt("Enter your move: rock, paper, scissors")
+  //Converts player input into a number between 0 and 2 with the scheme:
+  //rock = 0
+  //paper = 1
+  //scissors = 2
+  switch (playerinput) {
+    case 'rock':
+      return '0';
```

Comment:
You are returning strings here, not integers, which means you can't compare them mathematically in the way you attempt to do below.